baker/DeployUtils.ts

```diff
@@ -43,14 +43,15 @@ const bakeAndDeploy = async (
     }
 }
 
-export const tryBake = async () => {
+export const bake = async () => {
     const baker = new SiteBaker(BAKED_SITE_DIR, BAKED_BASE_URL)
     try {
         await baker.bakeAll()
+        baker.endDbConnections()
```

Comment:
that could live in a `finally` block?

Reply:
Sorry I missed this.

It could, but a `finally` block implies that we'll get to it regardless whether an error throws. In this case, since we rethrow the error in `catch`, I thought it's misleading to have a `finally` block, because it will never be reached from `catch` only from `try`, and in that case, it's better to only have it in the `try` block.

Reply:
Actually it also makes sense to be in `finally`